boxoffice/views/order.py

```diff
@@ -438,6 +439,11 @@ def payment(order):
                 invoice_no=order.invoice_no,
             ),
         )
+        if app.config['TELEGRAM_APIKEY'] and app.config['TELEGRAM_CHATID']:
+            send_telegram_message.queue(
+                buyer_fullname=order.buyer_fullname,
+                line_item_title=order.line_item.item.title,
```

Comment:
`order.line_items` is a list. Don't have this extraction logic here. Pass just `order.id` to the background job, and let is extract the data it needs.